Documentation/faq.md

```diff
@@ -38,6 +64,10 @@ It is recommended to have an odd number of members in a cluster. An odd-size clu
 
 Adding a member to bring the size of cluster up to an even number doesn't buy additional fault tolerance. Likewise, during a network partition, an odd number of members guarantees that there will always be a majority partition that can continue to operate and be the source of truth when the partition ends.
 
+#### Deploy cross-region, cross data center?
+
```

Comment:
Yes, you can. Deploying etcd across regions add resilience to etcd by putting mebers into different failure domain. The cost is higher request latency. If a client in data center A talks to an etcd member in data center B, it might see higher latency. Also internally etcd replicates data across all its peers, so the latency increases.

You might also want to tune election timeout higher to avoid false positive result of health checking. See ...